src/auth/auth.service.ts

```diff
@@ -1,16 +1,16 @@
 import { Injectable } from '@nestjs/common';
-import { UsersService } from '../users/users.service';
 import { JwtService } from '@nestjs/jwt';
+import { UserService } from '../user/user.service';
 
 @Injectable()
 export class AuthService {
   constructor(
-    private usersService: UsersService,
+    private userService: UserService,
     private jwtService: JwtService,
```

Comment:
I think we wanted to make these `readOnly`?

Reply:
Yes, forgot to change it.

Reply:
Cleaned up: https://github.com/gipfeli-io/gipfeli-api/pull/16